--- ginga-0.13.6/configure.orig	2014-05-12 20:07:41.469044191 -0300
+++ ginga-0.13.6/configure	2014-05-12 20:07:55.713044494 -0300
@@ -17943,33 +17943,33 @@
     pkg_cv_FFMPEG_CFLAGS="$FFMPEG_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_FFMPEG_CFLAGS=`$PKG_CONFIG --cflags "libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101" 2>/dev/null`
+  pkg_cv_FFMPEG_CFLAGS=`$PKG_CONFIG --cflags "libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -17981,33 +17981,33 @@
     pkg_cv_FFMPEG_LIBS="$FFMPEG_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_FFMPEG_LIBS=`$PKG_CONFIG --libs "libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101" 2>/dev/null`
+  pkg_cv_FFMPEG_LIBS=`$PKG_CONFIG --libs "libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -18028,50 +18028,50 @@
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        FFMPEG_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101" 2>&1`
+	        FFMPEG_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101" 2>&1`
         else
-	        FFMPEG_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "libavcodec >= 55.39.101
-  libavdevice >= 55.5.100
-  libavformat >= 55.19.104
-  libavutil >= 52.48.101
-  libavfilter >= 3.90.100
-  libpostproc >= 52.3.100
-  libswresample >= 0.17.104
-  libswscale >= 2.5.101" 2>&1`
+	        FFMPEG_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "libavcodecffmpeg >= 55.39.101
+  libavdeviceffmpeg >= 55.5.100
+  libavformatffmpeg >= 55.19.104
+  libavutilffmpeg >= 52.48.101
+  libavfilterffmpeg >= 3.90.100
+  libpostprocffmpeg >= 52.3.100
+  libswresampleffmpeg >= 0.17.104
+  libswscaleffmpeg >= 2.5.101" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$FFMPEG_PKG_ERRORS" >&5
 
 	as_fn_error $? "
 *** The following FFmpeg libraries are required:
-*** libavcodec >= 55.39.101
-*** libavdevice >= 55.5.100
-*** libavformat >= 55.19.104
-*** libavutil >= 52.48.101
-*** libavfilter >= 3.90.100
-*** libpostproc >= 52.3.100
-*** libswresample >= 0.17.104
-*** libswscale >= 2.5.101" "$LINENO" 5
+*** libavcodecffmpeg >= 55.39.101
+*** libavdeviceffmpeg >= 55.5.100
+*** libavformatffmpeg >= 55.19.104
+*** libavutilffmpeg >= 52.48.101
+*** libavfilterffmpeg >= 3.90.100
+*** libpostprocffmpeg >= 52.3.100
+*** libswresampleffmpeg >= 0.17.104
+*** libswscaleffmpeg >= 2.5.101" "$LINENO" 5
 elif test $pkg_failed = untried; then
      	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
 $as_echo "no" >&6; }
 	as_fn_error $? "
 *** The following FFmpeg libraries are required:
-*** libavcodec >= 55.39.101
-*** libavdevice >= 55.5.100
-*** libavformat >= 55.19.104
-*** libavutil >= 52.48.101
-*** libavfilter >= 3.90.100
-*** libpostproc >= 52.3.100
-*** libswresample >= 0.17.104
-*** libswscale >= 2.5.101" "$LINENO" 5
+*** libavcodecffmpeg >= 55.39.101
+*** libavdeviceffmpeg >= 55.5.100
+*** libavformatffmpeg >= 55.19.104
+*** libavutilffmpeg >= 52.48.101
+*** libavfilterffmpeg >= 3.90.100
+*** libpostprocffmpeg >= 52.3.100
+*** libswresampleffmpeg >= 0.17.104
+*** libswscaleffmpeg >= 2.5.101" "$LINENO" 5
 else
 	FFMPEG_CFLAGS=$pkg_cv_FFMPEG_CFLAGS
 	FFMPEG_LIBS=$pkg_cv_FFMPEG_LIBS
